assets/js/src/core/modules/perspectives/enums/nav-permission.ts

```diff
@@ -28,6 +28,7 @@ export enum NavPermission {
   UsersHidden = 'settings.users_hidden',
   Users = 'settings.users_users',
   Roles = 'settings.users_roles',
+  Redirects = 'settings.redirects',
```

Comment:
see: https://github.com/pimcore/pimcore/blob/f0998e09cb534c599c1e045dfdb1e6791bd60c03/bundles/SeoBundle/public/js/startup.js#L75C92-L75C101

```suggestion
  Redirects = 'extras.redirects',
```

Reply:
see https://www.figma.com/design/kTxowT29eCHwNh5NspDGGu/Pimcore-Components?node-id=12304-130526&p=f&t=aS8qvLKTIQ40sHFa-0

We now have an entirely new design for the Menu. 
Its one of the next tasks on my list to update all menu points.